packages/devextreme-react/README.md

```diff
@@ -1,23 +1,24 @@
-# DevExtreme React UI Components #
+# DevExtreme for React
 
-[![Build Status](https://api.shippable.com/projects/5a9532d976382b0800e7a53c/badge?branch=master)](https://app.shippable.com/github/DevExpress/devextreme-react)
-[![NPM](https://img.shields.io/npm/v/devextreme-react.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-react)
+[![NPM version](https://img.shields.io/npm/v/devextreme-react.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-react)
 
-This project allows you to use [DevExtreme widgets](http://js.devexpress.com/Demos/WidgetsGallery/) in [React](https://reactjs.org) applications.
+The `devextreme-react` package contains the [DevExtreme component suite](http://js.devexpress.com/Demos/WidgetsGallery/) for the [React](https://reactjs.org) framework.
 
-* [Documentation](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/)
-* [Technical Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/React/Light/)
-* [API Reference](http://js.devexpress.com/Documentation/ApiReference/)
-* [Live example](https://stackblitz.com/edit/devextreme-react)
+* [Documentation](https://js.devexpress.com/React/Documentation/Guide/React_Components/DevExtreme_React_Components/)
+* [Component Demos](https://js.devexpress.com/React/Demos/WidgetsGallery/Demo/DataGrid/Overview/React/Light/)
+* **Template Gallery**
+    * [Fluent theme](https://devexpress.github.io/devextreme-ui-template-gallery/fluent.blue.light/react/default/#/crm-contact-list)
+    * [Generic theme](https://devexpress.github.io/devextreme-ui-template-gallery/generic.light.compact/react/default/#/crm-contact-list)
+* [Sandbox example](https://stackblitz.com/edit/devextreme-react)
```

Comment:
let's get rid of this link

Reply:
React Application Templates https://js.devexpress.com/React/Documentation/Guide/React_Components/Application_Template/

Reply:
+ everything that applies for Angular